geomesa-hbase/geomesa-hbase-tools/bin/aws-utils/aws-bootstrap-geomesa-jupyter.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+# Bootstrap Script to install a GeoMesa Jupyter notebook
+#
+
+# Load common functions and setup
+if [ -z "${%%gmtools.dist.name%%_HOME}" ]; then
+  export %%gmtools.dist.name%%_HOME="$(cd "`dirname "$0"`"/../..; pwd)"
+fi
+
+function log() {
+  timeStamp=$(date +"%T")
+  echo "${timeStamp}| ${@}" | tee -a /tmp/bootstrap.log
+}
+
+if [[ "${1}" == "-h" || "${2}" == "--help" ]]; then
+  echo "Usage: ./aws-bootstrap-geomesa-jupyter.sh <jupyter_password>"
+fi
+
+# Verify that we are running in sudo mode
+if [[ "$EUID" -ne 0 ]]; then
+  log "ERROR: Please run in sudo mode"
+  exit
+fi
+
+user="ec2-user"
+
+JUPYTER_PASSWORD=$1
+if [[ -z "${JUPYTER_PASSWORD}" ]]; then
+  JUPYTER_PASSWORD="geomesa"
+  log "Using default password: geomesa"
+fi
+
+log "Installing Python 3.6"
```

Comment:
this may need to be parameterized to not just be 3.6?

Reply:
Maybe, but I was having problems with Jupyter, they updated some stuff so that it works on Python 3.4.4 but not 3.4.3...